### ISSUE TITLE
[Safari] Missing Visual Focus on "Add To Home Screen": Order Confirmation

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4328]
![Screen Shot 2019-08-29 at 7 31 38 PM](https://user-images.githubusercontent.com/15113219/63985245-a72d8d00-ca9c-11e9-88c3-7ca90fe9f311.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔
- Firefox: ✔ (note: focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
when i opened the link directly from slack, it opens (not sure which browser it's using).
And i don't get the "Add to Home" thingy on order confirmation page.
when I opened the link on safari directly, i can't even load the page, because on my phone, it's kinda blocking it, I don't get the certificate, so i can't accept it. I just get a blank page.
And I did have pwa enabled and started with ssl.

**Comment by Xymmer** [comment]
I tried it too and can't see the Add to Home
Can we get a special build that always displays add to home for testing purposes? a config would be ideal.

**Comment by developpeurweb** [comment]
It was probably `dev-com-17` since it was somehow a default last summer, but I don't get the **"Add To Home"** button anymore, nor I know the name of the feature that places a button in the Order Confirmation.
Therefore, the issue in this ticket cannot be reproduced until we identify the feature and set it up in a server.

**Comment by developpeurweb** [comment]
Reproducing this issue is totally beyond my area of expertise due to server configurations, @Platonn shed some light on it as per screenshot below, so we need a dev to set this up first, therefore unassigning myself from this ticket until that happens.

![Screen Shot 2020-03-02 at 1 24 15 PM](https://user-images.githubusercontent.com/15113219/75705349-2a68af00-5c89-11ea-8ce4-ca6cd12637c4.png)

cc @KateChuen @Xymmer @kacperknapik
